commit 29af5136171e7f2b64266ebecf8b2cd81fa78c1b
Author: J. R. Okajima <hooanon05@yahoo.co.jp>
Date:   Thu Jul 9 23:39:06 2009 +0900

    aufs: support linux-2.6.29.4, ->page_mkwrite() paramter
    
    Convert the type of the paramter.
    
    Signed-off-by: J. R. Okajima <hooanon05@yahoo.co.jp>
diff --git a/fs/aufs/f_op.c b/fs/aufs/f_op.c
index 28a6965..1fead86 100644
--- a/fs/aufs/f_op.c
+++ b/fs/aufs/f_op.c
@@ -419,7 +419,7 @@ static int aufs_fault(struct vm_area_struct *vma, struct vm_fault *vmf)
 	return err;
 }
 
-static int aufs_page_mkwrite(struct vm_area_struct *vma, struct page *page)
+static int aufs_page_mkwrite(struct vm_area_struct *vma, struct vm_fault *vmf)
 {
 	int err;
 	static DECLARE_WAIT_QUEUE_HEAD(wq);
@@ -434,7 +434,7 @@ static int aufs_page_mkwrite(struct vm_area_struct *vma, struct page *page)
 
 	mutex_lock(&finfo->fi_vm_mtx);
 	vma->vm_file = h_file;
-	err = finfo->fi_h_vm_ops->page_mkwrite(vma, page);
+	err = finfo->fi_h_vm_ops->page_mkwrite(vma, vmf);
 	au_reset_file(vma, file);
 	mutex_unlock(&finfo->fi_vm_mtx);
 	wake_up(&wq);
